sdk/keyvault/keyvault-keys/test/public/keyClient.hsm.spec.ts

```diff
@@ -71,5 +72,84 @@ onVersions({ minVer: "7.2" }).describe(
         );
       });
     });
+
+    onVersions({ minVer: "7.3-preview" }).describe("releaseKey", () => {
+      const releasePolicy = {
```

Comment:
Right now I have the releaseKey method accept a byte array containing json bytes for this policy. I was thinking maybe we should offer a method that takes a JSON blob and does the encoding in the convenience layer but I don't know yet how customers will use this so I left it as is.

Reply:
It might not always be JSON, so resolving this